mqbench/observer.py

```diff
@@ -93,6 +94,8 @@ def _calculate_qmin_qmax(self) -> Tuple[int, int]:
                 quant_min, quant_max = 0, qrange_len - 1
             if self.reduce_range:
                 quant_min, quant_max = quant_min // 2, quant_max // 2
+            if self.not_calc_quant_min_max:
```

Comment:
Is the "not_calc_quant_min_max" necessary here?

Reply:
Yes, it's necessary. It is used to set [0, 127] when `qsheme.dtype` is `torch.int8`, which would be corrected as symetric interval if adopt previous settings。

Reply:
why [0, 127] not [-128, 127] ?

Reply:
Maybe we can also directly use https://github.com/ModelTC/MQBench/blob/40b6d43f8f98d6573cf81c09027320c02f11fac5/mqbench/observer.py#L94

Reply:
But it can not reach this line, because it satisfy the requirement of `is_symmetric_quant` when its `qsheme.dtype` is `torch.int8`